style1.css

```diff
@@ -0,0 +1,235 @@
+/* Tablet+ */
+body {
+background: linear-gradient(90deg, #83EB94 0%, #47D95E 100%);
+min-width: 768px;
+margin: 4px;
+display: block;
+}
+
+.container {
+width: 768px;
+height: 1024px;
+margin-left: auto;
+margin-right: auto;
+}
+
+/* coming-soon-ribbon */
+.coming-soon {
+position: absolute;
+left: 83.85%;
+right: 0%;
+top: 0.1%;
+bottom: 81.45%;
+
+background: url(coming-soon-ribbon.png) no-repeat; 
+}
+
+/* Header */
+
+header {
+overflow: visible;
+}
+
+.header_image {
+margin: 64 40.8 652 40.8;
+
+}
+
+h1 {
+margin: 0%;
+height: 100px;
+width: 768px;
+
+font-family: 'Lato', sans-serif;
+font-style: normal;
+font-weight: 300;
+line-height: normal;
+font-size: 60px;
+text-align: center;
+letter-spacing: 5px;
+
+color: #FFFFFF;
+
+mix-blend-mode: normal;
+}
+
```

Comment:
To samo, stylowanie po tagach... Dodatkowo - czcionka Lato jest domyślną czcionką dla strony. Warto to tu umieszczać? Pewnie za chwilę w kodzie znajdę ponownie cały blok :D